tests/test_pubkey_cmd.py

```diff
@@ -1,13 +1,16 @@
-def test_get_public_key(cmd):
-    pub_key, chain_code = cmd.get_public_key(
+from boilerplate_client.boilerplate_cmd import BoilerplateCommand
+
+def test_get_public_key(backend, firmware):
+    client = BoilerplateCommand(backend)
+    pub_key, chain_code = client.get_public_key(
         bip32_path="m/44'/0'/0'/0/0",
         display=False
     )  # type: bytes, bytes
 
     assert len(pub_key) == 65
     assert len(chain_code) == 32
 
-    pub_key2, chain_code2 = cmd.get_public_key(
+    pub_key2, chain_code2 = client.get_public_key(
         bip32_path="m/44'/1'/0'/0/0",
         display=False
     )  # type: bytes, bytes
```

Comment:
I think we could also add reference Speculos seed expected value.
Something like that: https://github.com/LedgerHQ/app-eos/blob/develop/tests/functional/test_eos.py#L63
What do you think about it?

Reply:
Done in better using a new ragger module :)